libraries/api-extractor-model/src/model/ApiEnumMember.ts

```diff
@@ -25,6 +25,15 @@ export interface IApiEnumMemberOptions
   initializerTokenRange: IExcerptTokenRange;
 }
 
+/**
+ * Sort options for enum members in api reports and markdowns
+ * @public
+ */
+export enum EnumMemberOrder {
+  byName = "by-name",
+  preserve = "preserve"
```

Comment:
Looking at other enums in the RushStack projects, the convention seems to be capitalized camel case. So `ByName` instead of `byName`.